src/python/torch_radon/radon.py

```diff
@@ -169,14 +169,14 @@ def filter_sinogram(
         pad = padded_size - size
         padded_sinogram = torch.nn.functional.pad(sinogram.float(), (0, pad, 0, 0))
 
-        sino_fft = torch.fft.rfft2(padded_sinogram) / np.sqrt(padded_size)
+        sino_fft = torch.fft.rfft(padded_sinogram, norm="ortho")
 
         # get filter and apply
         f = self.fourier_filters.get(padded_size, filter_name, sinogram.device)
```

Comment:
I would suggest creating fourier filter `f` within `__init__` so we don't need to create it in an online way for each iteration. For example:
```Python
def __init__(self, ..., num_dets, filter_name="ramp"):
    self.filter_name = filter_name
    size = num_dets
    padded_size = max(64, int(2 ** np.ceil(np.log2(2 * size))))
    self.fourier_filter = FourierFilters().get(padded_size, filter_name, device='cpu')
    self.fourier_filter.requires_grad = False
    ...

def filter_sinogram(...):
    ...
    filtered_sino_fft = sino_fft * self.fourier_filter.to(sino_fft.device)
    ...
```

Reply:
>  in an online way for each iteration

What do you mean? The filter is cached in the FourierFilter class using a Python dictionary, so that if the same filter is used multiple times, then it is reused.

Reply:
Oops that's my mistake, didn't notice the cache. Given that, I think this part looks great! 👍